specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2018-10-01/StorageSubSystem.json

```diff
@@ -91,6 +91,12 @@
           },
           {
             "$ref": "../2016-05-01/Fabric.json#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "../2016-05-01/Fabric.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "../2016-05-01/Fabric.json#/parameters/SkipParameter"
```

Comment:
referring files from different api version may be a bit dangerous and have some unexpected consequences.
i think it would be better to move these common definitions to a separate common.json file.
I see that it was done before, but maybe it's a good time to fix it?

Reply:
agree, @viananth lets make a copy